doc/cli.rst

```diff
@@ -0,0 +1,14 @@
+.. Copyright (c) 2016-2020 The Regents of the University of Michigan
+.. This file is part of the General Simulation Data (GSD) project, released
+.. under the BSD 2-Clause License.
+
+gsd command line interface
+==========================
+
+**GSD** provides a command line interface for rapid inspection of files from the command line.
+
+Package contents

```

Comment:
Remove this heading. The way the docs render, the reader doesn't really know or need to know that it comes from a python package.